Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -55,8 +55,7 @@ file that was distributed with this source code.
                         'code':     sonata_admin.field_description.associationadmin.code,
                         'objectId': sonata_admin.field_description.associationadmin.id(sonata_admin.value),
                         'uniqid':   sonata_admin.field_description.associationadmin.uniqid
-                        })
-                    )}}
+                        })) }}
```

Comment:
It's not php, so nothing to do with the PSR, but indention means that just like you indent content inside a hash, like this : 

```
{% set my hash = {
    'item1',
    'item2',
    'item3'
} %}

you have to do it no matter where the hash is (in our case, inside a function call).